tests/schema/test.formats.js

```diff
@@ -1,73 +1,134 @@
-import { isRelativeURL, isValidVersionString } from 'schema/formats';
-
-describe('formats.isValidVersionString', () => {
-
-  var validVersionStrings = [
-    '0.1.12dev-cb31c51',
-    '1.0',
-    '1.0.0beta2',
-    '2.10.2',
-    '3.1.2.4567',
-    '3.1.2.65535',
-    '4.1pre1',
-    '4.1.1pre2',
-    '4.1.1dev-abcdef1',
-    '4.1.1.2pre3',
-  ];
-
-  var invalidVersionStrings = [
-    2,
-    '123e5',
-    '1.',
-    '.',
-    'a.b.c.d',
-    '1.2.2.2.4',
-    '01',
-    '1.000000',
-    '2.99999',
-    '3.65536',
-    '1.0.0-beta2',
-  ];
-
-
-  for (let validVersionString of validVersionStrings) {
-    it(`should find ${validVersionString} to be valid`, () => {
-      expect(isValidVersionString(validVersionString)).toBeTruthy();
+import {
+  isAbsoluteUrl,
+  isAnyUrl,
+  isSecureUrl,
+  isStrictRelativeUrl,
+  isValidVersionString,
+} from 'schema/formats';
+
+describe('formats', () => {
+  describe('isValidVersionString', () => {
+    var validVersionStrings = [
+      '0.1.12dev-cb31c51',
+      '1.0',
+      '1.0.0beta2',
+      '2.10.2',
+      '3.1.2.4567',
+      '3.1.2.65535',
+      '4.1pre1',
+      '4.1.1pre2',
+      '4.1.1dev-abcdef1',
+      '4.1.1.2pre3',
+    ];
+
+    var invalidVersionStrings = [
+      2,
+      '123e5',
+      '1.',
+      '.',
+      'a.b.c.d',
+      '1.2.2.2.4',
+      '01',
+      '1.000000',
+      '2.99999',
+      '3.65536',
+      '1.0.0-beta2',
+    ];
+
+
+    for (let validVersionString of validVersionStrings) {
+      it(`should find ${validVersionString} to be valid`, () => {
+        expect(isValidVersionString(validVersionString)).toBeTruthy();
```

Comment:
from recent discussions with @muffinresearch this should probably be `expect(...).toEqual(true)` to unify our asserts a bit and to avoid testing for anything else but the expected `true` value.